### ISSUE TITLE
Unread chapter  count on card. 

### ISSUE THREAD

**Issue by digiwombat** [raulhaag/MiMangaNu#180]
Hiding manga with no unread chapters is great, but I would love to have and unread count on the card in the library view. Would provide an easy way to decide where to spend my reading time if it's been a few weeks. Also, would be very convenient for folks who don't want to hide read manga.

**Comment by Dostoi** [comment]
Only as an option maybe, to avoid having too many things on the manga page.

**Comment by digiwombat** [comment]
Agreed there. Some people like it super minimal. I love having as much information as I can get.
